### ISSUE TITLE
Remove the underlays of the upper and lower menu items

### ISSUE THREAD

**Issue by sepulzera** [teeworlds/teeworlds#1791]
The new menu in general looks crisp and clean. The elements are designed flat, with much space.
But what really bothers me are those dirty looking semi-transparent underlays from the upper and lower menu items to the edge of my screen.

For me, it *looks* like a bug, like if my screen kept continuing drawing the boxes. I don't think it should be there.

![screenshot_ingame_buttons_underlay](https://user-images.githubusercontent.com/43857716/49510645-2a60e000-f889-11e8-9eed-9c2731db3eb5.jpg)

**Comment by Dune-jr** [comment]
I agree it looks a bit... dirty? I'm not sure what kind of effect is supposed to be achieved there.

**Comment by Zatline** [comment]
it's part of the design lol

**Comment by Zatline** [comment]
it's part of the new ui design very simple! start menu has these gradients so does the rest of the menu.

**Comment by sepulzera** [comment]
To be honest, I do not see any relation to the main menu here. The gradients on the main screen are large enough to create a visual feeling to be actual non-interactive elements.

![screenshot_main_menu](https://user-images.githubusercontent.com/43857716/49546058-aab23000-f8df-11e8-89bf-a3207e2b0dd7.jpg)

The main menu is the main content on this screen. The remaining is just a background. On this screen the main content is connected through those "gradients" / underlaying, semi transparent box.
For me, the main content is splitted into two parts by using those gradients. THIS is a design choice and I accept it. (Although imho at least the bottom gradient looks very weird, as it is grouping only one element.)


Now let us take a look at the serverbrowser.

![screenshot_serverbrowser](https://user-images.githubusercontent.com/43857716/49547034-fa91f680-f8e1-11e8-9b22-5670572adf2f.jpg)

In the middle there is the main content, the server browser. All those elements are put together with a semi-transparent, non-gardient box.
On the upper, there is a tab menu with two tabs. Both tabs are grouped via this gradient thingy.
On the lower, there is the back button. Again, a single element grouped via gradient.
Additionally, on the lower right are the refresh and connect buttons, both grouped via this gradient thingy.

**Putting things together**
After analyzing this design choice in particular, I come to the following conclusion:
1. The gradients are used as group boxes. This is great, as it helps the user to identify related elements quickly.
2. The gradients are barely visible, they are way too transparent. Therefore, they aren't clearly noticeable as group boxes. This is a design flaw and should be fixed.
3. It does not convey any information that the gradients are tied to the upper and lower screen boarder. More than that, I am not quite sure if it really is your intention. Those gradients still look like a display bug for me.
4. If you want to align the buttons to the upper and lower screen corner, why wouldn't you just put them there? For now, there is plenty of wasted screen space, filled up with gradients with no information at all, but covering the background themes.

**Comment by Zatline** [comment]
1. indeed
2. they shouldn't be distractig either it's a good inbetween just as it is
3. it was my intention yes and it still is and I'm probably going to be extremly upset if I see anything screwed with it. I think it looks really nice, very uniqie in fact, it makes this UI stand out and it is cosistent throughout the whole UI, except for what hasn't been revamped yet. It's actually one of my most favorite design coises for the UI, I don't quite get how u see a bug there ...
4. buttons sitting right at the edges / borders of the screen what? now that's boring and actual waste of space! srsly space needs to be used nice and evenly and that's what it is right now, it's an initial part of giving off that modern vibe that tw lacked before, there is nothing wrong here!

**Comment by sepulzera** [comment]
Zatline, you did an amazing job on this design. It indeed looks modern with its simple and flat design. It is clean and well structured. Overall it is a rock solid base to build on.

But UI design is difficult. No software design was perfect on its first draw. And I am pretty sure you did much more than that. Nevertheless, saying that you don't want so see any improvements is toxic for future progress. And in my opinion there is quite some room for improvements. Sometimes small changes make huge differences!

Therefore, I suggest to move this discussion (its main point) to the forum of teeworlds.com. To gather feedback from the actual users. The developers opinion shouldn't matter most, at least in regard of UX.

**Comment by Dune-jr** [comment]
My two cents:

- I think github is still the best place to talk about UX development. Although everyone's input should be welcome, there are always some technical aspects best discussed here.

- Whenever you design anything in Teeworlds, you have to accept that it's going to get modified/upgraded at some point, *especially* the UI. Same goes for code, there is no IP left once submitted but authorship.

**Comment by 1stDecadeTW** [comment]
I like how the pause menu look with shadow gradient n° 3:
![screenshot_1](https://user-images.githubusercontent.com/44511917/49607931-02db4780-f95d-11e8-92c8-504683f57b0f.jpg)
![screenshot_2](https://user-images.githubusercontent.com/44511917/49607933-02db4780-f95d-11e8-9977-b9d29026e5fc.jpg)
![screenshot_3](https://user-images.githubusercontent.com/44511917/49607934-02db4780-f95d-11e8-824e-b617675b3df3.jpg)

**Comment by Zatline** [comment]
no fan of any of these, especially first that full box blur one :/
